### ISSUE TITLE
A10 (iPad 7) sep pwn not working

### ISSUE THREAD

**Issue by nick-botticelli** [checkra1n/pongoOS#137]
I have attempted to run blackbird exploit (`sep pwn`) on j171ap (A10 T8010) however I am met with a familiar "Registers are locked" and am unable to decrypt kbags. I receive the same message with `sep tz0_blackbird`.

It was my understanding that A10 was supported, yet I have tried running `sep pwn` immediately once booted into PongoOS and have received the same error message (both with PongoOS shipped with checkra1n 0.12.4 and built by myself on 4c9b7541).

**Comment by Siguza** [comment]
The trustzone unlock patch is only applied on iOS 14.x. It sounds like you're below 14.0?

**Comment by nick-botticelli** [comment]
> The trustzone unlock patch is only applied on iOS 14.x. It sounds like you're below 14.0?

Yes, I am (13.4.1). I assume then it isn't compatible (or is it just not enabled)?
